docs/installation.md

```diff
@@ -1,18 +0,0 @@
-# Installation
-
-The recommended way to install the Lightning-CLI is as a _global_ package.
-You can install the latest version of the Lightning-CLI by running the following command:
-
-```bash
-npm install -g rdkcentral/Lightning-CLI
-```
-
-After this you will be able to call the Lightning-CLI anywhere on your machine via the `lng` command.
-
-Try it out by running:
-
-```bash
-lng -V
-```
-
-This should print out which version of the Lightning-CLI you have installed.
```

Comment:
I think we should maintain this content somewhere ..

Reply:
Installation folder & file added, updated CLI main topic with Installation option.